tests/gui/theme_test.py

```
@@ -0,0 +1,107 @@
+import unittest
+
+os.environ.setdefault("QT_QPA_PLATFORM", "offscreen")
+
+try:
+    from PyQt5.QtCore import Qt
+    from PyQt5.QtGui import QColor, QPalette
+    from PyQt5.QtWidgets import QApplication
+    from polyhost.gui import theme
+    _APP = QApplication.instance() or QApplication([])
+    _IMPORT_ERR = None
+except Exception as e:  # pragma: no cover - no Qt platform available
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)